src/collections/collection.ts

```diff
@@ -255,3 +267,11 @@ export class Collection {
   }
 
 }
+
+export class StargateMongooseError extends Error {
+  message: string;
+  constructor(message: any) {    
+    super(`Operation failed with the following error: ${message}`);
```

Comment:
Same comments as #53. Would be great to report that this was a `deleteMany` that failed, and add a property that contains the command that was sent to the server.

Reply:
Makes sense, ll add.